tests/devices/unit_tests/test_controllers.py

```diff
@@ -0,0 +1,11 @@
+from unittest.mock import Mock
+
+from dodal.devices.controllers import (
+    ConstantDeadTimeController,
+)
+
+
+def test_constant_dead_time_controller_returns_constant():
+    constant_deadtime = 2.4
+    controller = ConstantDeadTimeController(driver=Mock(), deadtime=constant_deadtime)
+    assert controller.get_deadtime(exposure=Mock()) == constant_deadtime
```

Comment:
Can we paramterise this to show that the deadtime is independent on exposure?

Just check get_deadtime for `exposure=[0.001, 0.01, 0.1, 1, 10, 100]` etc.